kotlin/internal/jvm/android.bzl

```diff
@@ -119,4 +120,5 @@ def kt_android_local_test(
         manifest_values = manifest_values,
         tags = kwargs.get("tags", default = None),
         testonly = testonly,
+        exec_properties = exec_properties,
```

Comment:
Wouldn't we want to also pass this to the rest of the targets that are created here so that the `exec_properties` are consistent for all targets?

Reply:
Makes sense 👍 I think that your comment should be addressed in https://github.com/bazelbuild/rules_kotlin/pull/810/commits/9c922b529e2de2fb25ede00d8c280444d2487e32

Reply:
https://github.com/bazelbuild/rules_kotlin/pull/810/files#diff-686482fb7b1546f7ba4e615a0e5f45354cf1f74934c3a31165571540ff96796dR68 the `kt_android_library` macro could use this property as well so that the two macros provide consistent attributes.

Reply:
OK, this time it should _really_ be addressed in https://github.com/bazelbuild/rules_kotlin/pull/810/commits/9601771bab57aa03c34d9d1fff2266f8275d47d0